Data ingestion using SINGER.py

```diff
@@ -1,6 +1,33 @@
 #importing package to create a JSON
 import singer as sg
+#practicing the dict() constructor
+nums = dict(x=1,y=2)
+print(type(nums))
+print(nums)
+
+#creating a dictionary using iteratable
+nums1 = dict([('x',5),('y',4)])
+print(nums1)
+
+nums2 = dict([('x',3),('y',6)],z=7)
+print(nums2)
 
+nums3 = list(zip(['x','y','z'],[1,3,5]))
+print(nums3)
+print(dict(nums3))
+
+nums4 = dict(zip(['p','q','r'],[6,8,0]))
+print(nums4)
+
+qwe = ('q','t','z')
+asd = {(1,2,3),(4,5,6),(7,8,9)}
+print(asd.pop())
+print(dict(zip(qwe,asd.pop())))
+
+#understood the nuances of using pop() method with the zip() method. the difference in the way the two iterators get mapped.
```

Comment:
Good explanation. make sure you track what you learnt on the calendar. Also add it to your dictionary notes.